data/json/recipes/recipe_food.json

```diff
@@ -9234,6 +9237,77 @@
   ]
 },{
   "type" : "recipe",
+  "result": "jar_sauerkraut_pickled",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_VEGGI",
+  "skill_used": "cooking",
+  "difficulty": 5,
+  "time": 30000,
+  "reversible": false,
+  "autolearn": true,
+  "qualities" : [
+    {"id":"CUT","level":1,"amount":1},
+    {"id":"COOK","level":3,"amount":1}
+  ], "tools": [
+    [
+      [ "hotplate", 20 ],
+      [ "char_smoker", 1 ],
+      [ "toolset", 20 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "water", 1 ],
+      [ "water_clean", 1 ],
+      [ "salt_water", 1 ],
+      [ "saline", 5 ]
+    ],
+    [
+      [ "jar_glass", 1 ]
+    ],
+    [
+      ["irradiated_lettuce", 2],
+      ["lettuce", 2]
+    ],
+    [
+      [ "vinegar", 1 ]
```

Comment:
No vinegar; it's not sauerkraut.

You use cabbage and salt, and let it ferment - check the flask of yeast for an example of how to make a fermentation food recipe.